src/com/dd/controller_util/controller/RunningGameController.java

```diff
@@ -76,7 +76,7 @@ private void handleSaveButtonAction(ActionEvent event) throws FileNotFoundExcept
 		PrintStream toGsonFile = new PrintStream(gsonFile);
 		toGsonFile.println(new Gson().toJson(gameState));
 		toGsonFile.close();
-		output.appendText("The game has been saved as " + gameState.getName() + ".json.");
+		output.appendText("The game has been saved as " + gameState.getName() + ".json.\n");
```

Comment:
I think we don't need to have the .json at the end. Just the name of the game is fine